app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -135,7 +135,7 @@
     <% end %>
     <% unless @edition_taxons.any? %>
       <%= render "govuk_publishing_components/components/warning_text", {
-        text: "Please add a tag before publishing"
+        text: "Add topic tags before you can publish this document."
```

Comment:
I'm not sure this reads perfectly. Should it say "You must add topic tags before..."?